### PR TITLE
Feature/nodes setup readme

### PR DESCRIPTION
**Pull request by prakhar-cs**

### CODE REVIEW

File: NODES_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+#### Obtaining the certificate
+
+* SSH to the host machine 
+* Run the following command
+````
+docker run --rm -it -v "/root/letsencrypt/log:/var/log/letsencrypt" -v "/var/www/html/shared:/var/www/" -v "/etc/letsencrypt:/etc/letsencrypt" -v "/root/letsencrypt/lib:/var/lib/letsencrypt" lojzik/letsencrypt certonly --webroot --webroot-path /var/www --email EMAIL -d domain
+```` 
+
+### Gaia Hub 
+
```
Comment (ramendra11) [inline NODES_SETUP.md]: please add steps for domains creations.

File: NODES_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+####Setup
+
+* Make a folder `$HOME/hub` and copy the below configuration to `$HOME/hub/config.json` and add in your desired configuration. 
+````
+{
+  "port": 3000,
+  "driver": "",
+  "bucket": "",
```
Comment (ramendra11) [inline NODES_SETUP.md]: is this supposed to be left blank? what is the desired config?

File: NODES_SETUP.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    proxy_set_header Host $http_host;
+    proxy_redirect off;
+    proxy_pass http://gaia_hub;
+    break;
+  }
+}
+````
+
+- `sudo rm /etc/nginx/sites-enabled/default && sudo ln $HOME/hub/nginx.conf /etc/nginx/sites-enabled/default`
+- Finish `certbot` setup and cert generation
```
Comment (ramendra11) [inline NODES_SETUP.md]: Some more details regarding SSL should be given

**Comment by quixote911** [comment]
1. Clearly explain prerequisites - S3 Bucket, Domain1, Domain2
2. Then get them to setup the whole thing end-to-end **without HTTPS**
3. Then explain that HTTPS is requires so you need a certificate. You can give the certbot example then.

**Pull request closed without merging**